lib/uaa/info.rb

```diff
@@ -101,8 +101,13 @@ def validation_key(client_id = nil, client_secret = nil)
   # signature of these tokens, refer to the +kid+ header of the JWT token. The
   # +validation_keys_hash+ method returns a hash of all currently valid
   # verification keys, indexed by +kid+. To retrieve symmetric keys as part of
-  # the result, client credentials are required.
-  # @param (see Misc.server)
+  # the result, client credentials are required with +uaa.resource+ scope.
```

Comment:
@drnic - if a call is made with no client credentials (or client credential with insufficient scope access) will it still succeed (minus the private keys)? (ie will it still receive the public keys?)

Reply:
Yes, the default behavior of `/token_keys` is to return the asymmetric public keys.

Reply:
I can't actually get the UAA to return symmetric keys using instructions in API docs https://github.com/cloudfoundry/uaa/issues/867